build/azure-pipelines.yml

```diff
@@ -92,11 +96,14 @@ stages:
               targets: coreBuild
               workingDirectory: src/Umbraco.Web.UI.Client
           - script: npm ci --no-fund --no-audit --prefer-offline
-            workingDirectory:  src/Umbraco.Web.UI.New.Client
+            workingDirectory: src/Umbraco.Web.UI.New.Client
             displayName: Run npm ci
           - script: npm run build:for:cms
             displayName: Run New Backoffice Build
             workingDirectory: src/Umbraco.Web.UI.New.Client
+          - script: npm run build:for:npm
```

Comment:
Is this not missing a condition? to only to it when npmDeploy=true

Reply:
Yes, we could do that, although this way it always publishes the artifact. If we can live with this extra build, we will always have the same amount of artifacts, which is useful for the nightly release pipelines as well as debugging purposes.

In a future iteration we can make the build much faster, since we are now sure in which order the builds need to be run.